analytics.js

```diff
@@ -706,25 +765,61 @@ class Analytics {
           this.clientIntegrationObjects,
         );
 
-        // try to first send to all integrations, if list populated from BE
-        succesfulLoadedIntersectClientSuppliedIntegrations.forEach((obj) => {
-          try {
-            if (!obj.isFailed || !obj.isFailed()) {
-              if (obj[type]) {
-                const sendEvent = !this.IsEventBlackListed(rudderElement.message.event, obj.name);
-
-                // Block the event if it is blacklisted for the device-mode destination
-                if (sendEvent) {
-                  const clonedRudderElement = cloneDeep(rudderElement);
-                  obj[type](clonedRudderElement);
-                }
-              }
+        const intgWithoutTransformation = [];
```

Comment:
Let's refactor this whole logic to avoid repetition.

Reply:
Addressed.